# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gsl/k6_k7.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./ode-initval/rk8pd.c.orig  Sat Dec  1 15:10:29 2001
+++ ./ode-initval/rk8pd.c       Sat Dec  1 15:13:07 2001
@@ -30,6 +30,10 @@

 #include "odeiv_util.h"

+/* undefine k6 and k7 if set (here are variables with such a name) */
+#undef k6
+#undef k7
+
 /* Prince-Dormand constants */

 static const double Abar[] = {
--- ./ode-initval/rkck.c.orig   Tue Jun 12 13:12:36 2001
+++ ./ode-initval/rkck.c        Sat Dec  1 15:14:35 2001
@@ -30,6 +30,10 @@

 #include "odeiv_util.h"

+/* undefine k6 and k7 if set (here are variables with such a name) */
+#undef k6
+#undef k7
+
 /* Cash-Karp constants */
 static const double ah[] = { 1.0 / 5.0, 0.3, 3.0 / 5.0, 1.0, 7.0 / 8.0 };
 static const double b21 = 1.0 / 5.0;
--- ./ode-initval/rkf45.c.orig	2004-12-02 18:54:19.000000000 -0300
+++ ./ode-initval/rkf45.c	2005-02-08 16:54:27.700501964 -0300
@@ -32,6 +32,10 @@
 
 #include "odeiv_util.h"
 
+/* undefine k6 and k7 if set (here are variables with such a name) */
+#undef k6
+#undef k7
+
 /* Runge-Kutta-Fehlberg coefficients. Zero elements left out */
 
 static const double ah[] = { 1.0/4.0, 3.0/8.0, 12.0/13.0, 1.0, 1.0/2.0 };
